pipelines/jobs/configurations/jdk16_pipeline_config.groovy

```diff
@@ -10,6 +10,18 @@ class Config16 {
                 ],
                 configureArgs       : '--enable-dtrace'
         ],
+        
+        x64MacXL    : [
+                os                   : 'mac',
+                arch                 : 'x64',
+                additionalNodeLabels : 'macos10.14',
+                test                 : [
+                        nightly: [],
+                        weekly : ['sanity.openjdk', 'sanity.system', 'extended.system', 'sanity.perf']
+                ],
```

Comment:
```suggestion
                test                 : 'default',
```

Reply:
Container build on ppc has been adjusted as recommended. Good catch.

As for the tests, these were set to non-default b3ecause that makes them consistent with the non-XL equivalents on jdk16.

Reply:
Due to lack of comments I'm unable to say what's correct. So I've to page @smlambert for advice :disappointed: We really have to get better at communicating changes.

Reply:
Please use default.  We can adjust if we find that we do not have the machine capacity.

Reply:
Okie dokie. Making the change now.

Reply:
Done.